libs/sdk-backend-mockingbird/src/recordedBackend/types.ts

```diff
@@ -115,6 +118,42 @@ export type RecordedBackendConfig = IAnalyticalBackendConfig & {
         availableFacts?: (facts: ICatalogFact[]) => ICatalogFact[];
         availableDateDatasets?: (datasets: ICatalogDateDataset[]) => ICatalogDateDataset[];
     };
+
+    /**
+     * Optionally specify response for accessList
+     */
+    accessControl?: IAccessControl;
+
+    /**
+     * Optionally specify response for userGroup
+     */
+    userGroup?: IUserGroup;
+
+    /**
+     * Optionally specify response for users
+     */
+    users?: IUsers;
```

Comment:
what's the relation to the existing `user` here? please describe. ideally, group all the stuff related to users/groups & access so that it is next to each other. furthermore, it may make sense to do mild refactoring and stick these 4 props under some parent `userManagement` or something.. so that the main config is not so large; the second refactoring is not really crucial at this point.. just make sure things are close to each other within the config and that there is proper docs in place for the next person who tries to configure the recorded backend